python/dgl/graphbolt/sampling_graph.py

```diff
@@ -26,3 +38,51 @@ def num_nodes(self) -> Union[int, Dict[str, int]]:
             heterogenous graph.
         """
         raise NotImplementedError
+
+    @property
+    def num_edges(self) -> Union[int, Dict[str, int]]:
+        """The number of edges in the graph.
+        - If the graph is homogenous, returns an integer.
+        - If the graph is heterogenous, returns a dictionary.
+
+        Returns
+        -------
+        Union[int, Dict[str, int]]
+            The number of edges. Integer indicates the total edges number of a
+            homogenous graph; dict indicates edges number per edge types of a
+            heterogenous graph.
+        """
+        raise NotImplementedError
+
+    def copy_to_shared_memory(
+        self, shared_memory_name: str
+    ) -> "SamplingGraph":  # pylint: disable=invalid-name
```

Comment:
why this is a invalid-name? can't be just SamplingGraph?

Reply:
typo. removed.